spec/index.html

```diff
@@ -421,6 +569,91 @@ <h2>Fragment identifiers</h2>
       <h3>Examples</h3>
       <p class="ednote">FIXME</p>
     </section>
+    <section id="faq" class="informative" data-format="markdown">
+      **NOTE** REMOVE THIS SECTION BEFORE PUBLICATION.
+
+      <h3>FAQ</h3>
+
+      #### Why YAML-LD does not preserve comments?
```

Comment:
A parallel could also be made with other concrete syntaxes of RDF that *do* support comments (RDF/XML, Turtle).
I don't think that anybody ever suggested that these comments find their way into the graph...

Reply:
Yes, and reserializing these formats does loose comments, so it makes sense that YAML-LD comments would also be dropped. `rdfs:coment`, or similar, can be used in any serialization.

Reply:
Ok, I'll replace the FAQ above with some Turtle example. That's neat and clear.

Reply:
I replaced both FAQ and the text. It's simpler now. 84b05759a55e789819315764fd5b895531fbd403